.circleci/config.yml

```diff
@@ -1,42 +1,73 @@
 version: 2
 jobs:
-  build:
+  test:
     docker:
       - image: circleci/golang:1.10
     working_directory: /go/src/github.com/segmentio/aws-okta
     steps:
       - checkout
       - run:
-          name: Build
+          name: Test
           command: |
-            make govendor dist
+            make test
+
+  dist-linux:
+    docker:
+      - image: circleci/golang:1.10
+    working_directory: /go/src/github.com/segmentio/aws-okta
+    steps:
+      - checkout
+      - run:
+          name: Install nfpm, rpmbuild
+          command: |
+            sudo make -f Makefile.tools nfpm-debian rpmbuild-debian
+      - run:
+          name: Make distributables
+          command: |
+            make -f Makefile.release dist-linux
       - persist_to_workspace:
           root: .
           paths: ['dist/*']
 
-  release:
+  publish-linux:
```

Comment:
Yeah, I should add a comment. Basically because the darwin build depends on Keychain libs that are not freely distributable, so we need a real macOS box to build this on, which is a whole other thing :)

Reply:
https://github.com/segmentio/aws-okta/issues/81 for later

Reply:
Actually, I'm not sure this is correct. Chamber uses Keychain as well, and yet it builds fine... Hmmm

Reply:
Nvm, Chamber doesn't use keychain.